src/js/components/Loader.js

```diff
@@ -45,15 +45,15 @@ let typeMap = {
 class Loader extends React.Component {
   getDivs(length) {
     return Array.from({length}).map(function (_, index) {
-      return <div key={index} />;
+      return <div className="loader-element" key={index} />;
```

Comment:
I added a class to these elements to target them via CSS, rather than using the `> div` selector.